misc/sample_application/image_builder/create_image.sh

```diff
@@ -191,12 +197,12 @@ print_info "Guest image loop-mounted at /dev/${loop_dev}"
 try_step "Resizing guest root file system" resizing_guest_root /dev/mapper/${loop_dev}p1
 try_step "Mounting guest root file system at ${mount_point}" mount_filesystem /dev/mapper/${loop_dev}p1 ${mount_point}
 try_step "Preparing schroot configuration" create_schroot_config ${mount_point}
+try_step "Preparing apt config" dump_proxy
```

Comment:
`Preparing apt config` is not an accurate description of what this step is doing. Use `Extracting network proxy configurations` instead.

Reply:
Fix the description and force update the PR.